src/app/(Home)/(Domains)/domains/error.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+import { HeadingXSmall } from 'baseui/typography';
+import { styled } from 'baseui';
+import AlertIcon from 'baseui/icon/alert';
+
+// TODO: @assem.hafez extract error component to reusable error messages component
+const StyledContainer = styled('div', ({ $theme }) => {
+  return {
+    flex: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    gap: $theme.sizing.scale800,
+    padding: `${$theme.sizing.scale900} ${$theme.sizing.scale600}`,
```

Comment:
I don't think this will cause an issue in our case.  We aren't overriding any components and also the project is using a monolithic setup which prevents such issues.

Reply:
Yup, since this is just a container it should be okay
But if we are working with overrides, let's stick to longhand properties.